src/test/java/com/softserveinc/edu/ita/tests/OrderingPageSortingTest.java

```diff
@@ -0,0 +1,209 @@
+package com.softserveinc.edu.ita.tests;
+
+import com.softserveinc.edu.ita.dao_jdbc.domains.Order;
+import com.softserveinc.edu.ita.dao_jdbc.domains.User;
+import com.softserveinc.edu.ita.dataproviders.DataProviders;
+import com.softserveinc.edu.ita.enums.OrdersTable;
+import com.softserveinc.edu.ita.locators.HomePageLocators;
+import com.softserveinc.edu.ita.page_object.HomePage;
+import com.softserveinc.edu.ita.page_object.OrderingPage;
+import com.softserveinc.edu.ita.page_object.UserInfoPage;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import java.util.Comparator;
+import java.util.List;
+
+/**
+ * This class is used to test sorting actions in 'Ordering' table of 'Ordering' page.
+ */
+public class OrderingPageSortingTest extends TestRunner {
+    //Test sorting by Order Name
+    @Test(dataProvider = "getMerchandisers", dataProviderClass = DataProviders.class)
+    public void testOrderNameColumn(User user) {
+        HomePage homePage = new HomePage(driver);
+        UserInfoPage userInfoPage = homePage.logIn(user.getLogin(), user.getPassword());
+        OrderingPage orderingPage = userInfoPage.clickOrderingTab();
+
+        List<Order> tableFromView = orderingPage.getTableFromView();
+        tableFromView.sort(Comparator.comparing(Order::getOrderName));
+
+        orderingPage.clickOrdersTableColumn(OrdersTable.ORDER_NAME.toString());
+        List<Order> sortedTableByOrderNameAsc = orderingPage.getTableFromView();
+        orderingPage.clickOrdersTableColumn(OrdersTable.ORDER_NAME.toString());
+        List<Order> sortedTableByOrderNameDesc = orderingPage.getTableFromView();
+
+        for (int i = 0; i < tableFromView.size(); i++) {
+            Assert.assertTrue(sortedTableByOrderNameAsc.get(i).getOrderName().equals(tableFromView.get(i).getOrderName()),
+                    "Sorting by order name doesn't work.");
+        }
+
+        for (int i = 0, j = tableFromView.size() - 1; i < tableFromView.size(); i++, j--) {
+            Assert.assertTrue(sortedTableByOrderNameDesc.get(i).getOrderName().equals(tableFromView.get(j).getOrderName()),
+                    "Sorting by order name doesn't work.");
+        }
+    }
+
+    //Test sorting by Total Price
+    @Test(dataProvider = "getMerchandisers", dataProviderClass = DataProviders.class)
+    public void testTotalPriceColumn(User user) {
+        HomePage homePage = new HomePage(driver);
+        UserInfoPage userInfoPage = homePage.logIn(user.getLogin(), user.getPassword());
+        OrderingPage orderingPage = userInfoPage.clickOrderingTab();
+
+        List<Order> tableFromView = orderingPage.getTableFromView();
+        tableFromView.sort(Comparator.comparing(Order::getTotalPrice));
+
+        orderingPage.clickOrdersTableColumn(OrdersTable.TOTAL_PRICE.toString());
```

Comment:
move toString() into clickOrdersTableColumn() method
same goes for OrdersTable.ELEM.toString() calls